app/styles/component/_language-wikis-index.scss

```diff
@@ -0,0 +1,43 @@
+.language-wikis-index {
+  margin: 50px $article-horizontal-padding 0;
+  text-align: center;
+
+  &__intro {
+    font-size: $wds-font-size-l;
```

Comment:
https://fandomdesignsystem.com/base-styles/typography - please use classes not to bloat CSS size